src/tutorials/0007-verified-storage-on-filecoin/02.md

```diff
@@ -0,0 +1,23 @@
+---
+    title: "Proof-of-Replication (PoRep)"
+    type: "multiple-choice" # can be "text", "code", "file-upload" or "multiple-choice"
+---
+
+In a Proof of Replication, a storage miner proves that they are storing a _physically_ unique copy, or _replica_, of the data. This process happens just once, at the time the data is first stored by the miner and sealed into a sector.
+
+After a client negotiates a storage deal with a miner, the data transfer process begins. A client hires a miner to store a _piece_ of data, which might be a whole or partial file. Miners store these pieces from one or more clients in _sectors_, the fundamental storage unit used by Filecoin.
+
+## The pre-commit phase: Replication & CommD
+The replication of data happens in a _pre-commit_ phase, in which the storage miner places the replicated piece data into a sector. Each piece of data stored in Filecoin has a _payload CID_, similar to an IPFS CID, which is a cryptographic hash of the data itself. Since these CIDs are unique, identical CIDs guarantee identical content. No matter where data is stored or who's storing it, identical data will produce identical payload CIDs.
+
+Once a sector is full, a CommD (Commitment of Data) is produced, with an UnsealedSectorCID that represents the root node of all the payload CIDs contained in the sector. Most often, this will be an aggregate of pieces from a number of different clients.
+
+## The commit phase: Sealing & CommR
+
+In the _commit_ phase, a process called _sealing_ takes place, a sequential operation which gradually generates an encoding of the data for each miner, also known as a unique replica.
+
+During the sealing process, a _piece CID_ (or CommP) is generated for each piece of data stored in the sector. Unlike a payload CID, which is based on the data itself, a piece CID represents both the data and the deal. It's unique to the location where the data is stored and the miner who stored it. If two miners store the same piece of data for a client, it will have two different piece CIDs depending on who's storing it.
```

Comment:
```suggestion
During the sealing process, a _piece CID_ (or CommP) is generated for each piece of data stored in the sector. Unlike a payload CID, which is based on the data itself, a piece CID represents both the data and the deal, and is unique to the location where the data is stored and the miner who stored it. If two miners store the same piece of data for a client, it will have the same payload CID but two different piece CIDs.
```